lib/workspace.js

```diff
@@ -3,8 +3,25 @@ const request = require('request');
 
 const config = require('./config.js');
 const logger = require('./logger');
+const socketServer = require('./socket-server');
+
+const sqldb = require('@prairielearn/prairielib/sql-db');
+
+module.exports.init = function(callback) {
```

Comment:
We are setting `module.exports.init` here, but then below we are setting `module.exports = {...}` so we will lose the init function. Include `init` below if you want it. But maybe we are never calling it at the moment?